.github/workflows/bundle-size.yml

```diff
@@ -0,0 +1,27 @@
+name: Bundle-size
+
+on: [pull_request]
+
+jobs:
+  size:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: yarn
+
+      - uses: ./.github/actions/install-modules
+
+      - name: Link locale packages and install their dependencies
+        run: yarn bootstrap
+
+      - name: Build packages
+        run: yarn build
+
+      - uses: preactjs/compressed-size-action@v2
+        with:
+          pattern: '**/dist/**/*.{js,svg}'
```

Comment:
What do you mean by that? From what I understood, Vite would put esm/ cjs files in `dist` too, which should be matched by the globbing. I could transform this into `'**/dist/**/*.{mjs,js,svg}'`. `cjs` and `esm` wouldn't be file extensions I think?

Do you want only one report for `esm` OR `cjs`? I think I'd prefer that, but then you could also argue that we should include either `.production.{js}` OR `.development.{js}`.

Reply:
> What do you mean by that? From what I understood, Vite would put esm/ cjs files in dist too, which should be matched by the globbing.

I don't think they will be, for example, we produce an `index.js` which is ESM and `index.cjs` which is CJS. I also think it might be good to incl. `.d.ts` for when we introduce types, although this could be done later. The other examples like `mjs` were just to we capture more incase the build pipeline changes and we don't update the workflow.

Reply:
Alright, let's change the pattern to '**/dist/**/*.{cjs,js,svg}' - this should cover all cases you described, right? I'd prefer to add `.ts` files once we produce some.

Reply:
Sure! to be clear, it would just be `.d.ts` which is the `types` only 😄

Reply:
Sure :) Just out of curiosity: why would be interested in the size of the types in the end? From my understanding they would never reach the end-user.

Reply:
Suppose it would be good to understand the bundle size from a dev POV i.e. im downloading the design-system / strapi-admin app for local development – maybe that's overkill though?

Reply:
I'd say it is out of scope for the action because it may include not only the files in `dist` but other files depending on the release process.

I think tracking the installation size is another good initiative!